### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by haftamudesta**
This is my first hello Microverse project set up which is implemented in HTML and CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like crome
+## Author
+ [Githab](https://github.com/settings/profile)
+[linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+[tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
```
Comment (vechicin) [inline README.md]: Please change the generic platform names to your actual name/handle for that platform. It'll give you more visibility and make your documentation look more professional.

Additionally, follow the README template and include a header with your name before the links.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @haftamudesta ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing some of the previous reviewer's requirements.

 ## Required Changes ♻️

- [ ] Please **DO NOT OPEN NEW PULL REQUESTS** when requesting re-reviews. Keep this in mind for future reference as it may get your requests invalidated.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (haftamudesta) [reply README.md]: @vechicin , please could you make clear what is the mistake here. this project is a previously accepted project and according to microverse rule a previously accepted project should be accepted.

**Review by chrystalme — APPROVED** [review]
## Hi @haftamudesta , 

## Project Status ✔️  Approve
giphy


In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## [OPTIONAL REQUIREMENTS]
- Kindly use `index.html` as your default entry point for web based application rather than `hello.html`. This is the default entry point for web applications.
- Also, kindly begin your git commits with an infinitive verb

|Good Commit ✔️|Bad Commit ❌|
|---|---|
|Modify index.html|Modified index.html|
|Add test.md to .gitignore|Added test.md to .gitignore| 


Good luck!  🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**